docs/chapters/05_going-deeper.md

```diff
@@ -166,6 +166,92 @@ source <path-to-your-bash-file> && boost deploy -e production
 
 Now just let the magic happen, Booster will create everything for you and give you back your app ready to use URL. 🚀
 
+#### Azure synth command
+
+Azure provider implement the experimental **Booster** `synth` command. This command will generate [Terrafom](https://www.terraform.io/) templates from your code. It will also generate needed files to deploy your Booster application using [cdktf](https://learn.hashicorp.com/tutorials/terraform/cdktf).
+
+Running `synth` command, for example `boost synth -e production` will generate following files:
+
+* A file `cdktf.json`: A basic json file to deploy your application using `cdktf`
+* A folder `cdktf.out`: with the Terraform templates.
+
+Booster deploy command for Azure will deploy your application using the generated templates.  You don't need to run the `synth` command for deploy your application, the `deploy` command will generate the templates before deploy for you.
+
+Once you have the new files and folders generates you could use `cdktf` to deploy your application if you want to.
+
+#### Azure and CI/CD environments
+
+It is possible to deploy your Azure infrastructure using the [Terrafom](https://www.terraform.io/) templates generated by the `synth` command using Terraform executable. 
+
+To deploy a **Booster** application using the [Terrafom](https://www.terraform.io/) templates generated by the **Booster** `synth` command:
+
+1. Navigate to
+```shell
+> cd cdktf.out/stacks/<application name><environment name>
+```
+2. Run (only the first time)
+```shell
+> terraform init
+```
+3. Run
+```shell
+> terraform plan --out plan
+```
+4. Run
+```shell
+> terrafom apply "plan"
+```
+
+You could follow similar steps to integrate the Azure **Booster** deploys in your CI/CD environment.
+
+1. Navigate to
+```shell
+> cd cdktf.out/stacks/<application name><environment name>
+```
+2. Copy `functionApp.zip` to the destination folder
+```shell
+> cp functionApp.zip <destination>
+```
+
+After copying the files you should have the following structure:
+
+```text
+<application>
+├── cdktf.out
+│   └── stacks
+│       └── <application name><environment name>
+│           └── cdk.tf.json
+```
+
+Now deploy the template:
+
+1. Run (only the first time)
```

Comment:
I am not following here.... It seems we are repeating the steps here again.

Reply:
The first time we are talking about deploying from the user project point of view. You have everything in the root folder so you can do it. The next time is talking about what files you need to copy to your CI/CD environment to deploy (without the user project)

Reply:
ah!
ok thanks